INSTALLATION.md

```
@@ -13,44 +58,404 @@
-We provide a mostly automated way of setting up the development environment for talawa api using git, docker and visual studio code as shown in the following video:
+[![Talawa API Environment Setup - Development](https://img.youtube.com/vi/jz7koJIXqtk/0.jpg)](https://www.youtube.com/watch?v=jz7koJIXqtk)
 
-1. Click on the image to play the video
-2. The video is partially incorrect. Clone from this repo instead
-    - https://github.com/PalisadoesFoundation/talawa-api
+**Note: The video contains some inaccuracies.**
 
-[![Talawa API Environment Setup - Development](https://img.youtube.com/vi/jz7koJIXqtk/0.jpg)](https://www.youtube.com/watch?v=jz7koJIXqtk)
+   1. Please ensure you clone from the correct repository [Talawa API Repository](https://github.com/PalisadoesFoundation/talawa-api)
+   2. The correct branch to checkout is `develop-postgres`
+   3. `postgres-test` is the name of the test database in the docker-compose file. 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)